src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const cemetery = !century
+    ? people.filter(human =>
+      human.sex === 'm')
+
+    : people.filter(human =>
+      Math.ceil(human.died / 100) === century
+      && human.sex === 'm');
```

Comment:
Let's use more reliable check here, because century, theoretically, may have valid falsy value (0)

```suggestion
  const cemetery = century === undefined
    ? people.filter(human =>
      human.sex === 'm')
    : people.filter(human =>
      Math.ceil(human.died / 100) === century
      && human.sex === 'm');
```